src/Core/Datetime/DateFormatterTrait.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace drunomics\ServiceUtils\Core\Datetime;
```

Comment:
traits look good, but service-utils has the policy of 1 test case per trait. those are missing here and must be added first

Reply:
@fago tests added for both traits ... ok?